src/autoplex/auto/phonons/jobs.py

```diff
@@ -114,9 +275,10 @@ def complete_benchmark(  # this function was put here to prevent circular import
 
     for path in ml_path:
         suffix = Path(path).name
+        print(suffix)
         if suffix == "without_regularization":
             suffix = "without_reg"
-        if re.match(r"job_\d{4}-\d{2}-\d{2}-\d{2}-\d{2}-\d{2}-\d{6}-\d{5}", suffix):
+        if suffix not in ["phonon", "rattled"]:
```

Comment:
Old version only works for jobflow but not jobflow remote

Reply:
also my mistake for not thinking about the jobflow remote structure of jobs when implementing this.